scripts/validate_turns_metadata.csv

```diff
@@ -0,0 +1,16220 @@
+0,right

```

Comment:
Is this for the entire dataset or just for training/val ?

Reply:
Just for `val`. I will add two more for `train` and `train_full` as well.